config/locales/nl.yml

```diff
@@ -0,0 +1,210 @@
+---
+nl:
+  activerecord:
+    errors:
```

Comment:
Activerecord validatiefouten worden alleen in API's getoond, dus hoeven niet in het Nederlands